spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
```

Comment:
@Fryguy @jrafanie @chrisarcand Thoughts on this test?
It's indirectly testing ActiveSupport::Inflection `"bases".singularize -> "base"` fixed by https://github.com/ManageIQ/manageiq/pull/13632